be1-go/hub/organizer.go

```diff
@@ -197,6 +197,12 @@ func (o *organizerHub) createLao(publish message.Publish) error {
 	}
 
 	encodedID := base64.StdEncoding.EncodeToString(data.ID)
+	if _, ok := o.channelByID[encodedID]; ok {
+		return &message.Error{
+			Code:        -4,
```

Comment:
Shouldn't it be '-3' since it's a duplicate? [https://github.com/dedis/student_21_pop/blob/af8656538944c29f95dc5b1699716fa5656b2d1e/protocol/answer/negativeAnswer/error.json#L18](url)

Reply:
Nice catch! Fixing this